app/views/user_mailer/_form.html.haml

```diff
@@ -0,0 +1,7 @@
+=simple_form_for @simplemail, url: send_email_path do |f|
+  = f.input :addressee, label: "Do:"
```

Comment:
To pole powinno również być ukryte - adresat powinien być określany w kontrolerze